config/locales/en/views/cookies.yml

```diff
@@ -1,20 +1,86 @@
 ---
 en:
+  cookie_settings:
+    form:
+      cookies_off: 'Off'
+      cookies_on: 'On'
+      heading: Turn Google Analytics cookies on or off
+      hint: This is the same setting as set in the cookie banner
+      save_changes: Save changes
+    notification_banner:
+      preferences_set: You've set your cookie preferences. <a class="govuk-notification-banner__link" href="%{href}">Go back to the page you were looking at</a>.
```

Comment:
I would recommend appending `_html` to this key to mark it as hmtl_safe

Reply:
Done